docs/README-develop-best-practices.md

```diff
@@ -206,6 +206,12 @@ The following commands should be preferred over `show [feature]` commands since
 
 YAML allows various synonyms for `true` and `false` such as `yes` and `no`, but for consistency and readability (especially to users more familiar with Ruby than with YAML), we recommend using `true` and `false` rather than any of their synonyms.
 
+### <a name="yaml11">Y11: Use `_exclude` to return `nil` for unsupported properties.
+
+Some properties are only applicable to specific platforms. Rather than using `default_only` to specify an 'unconfigured' default like `''` or `false`, it is more accurate to return `nil` for a property that is not applicable at all. By returning `nil`, the property will not even appear in commands like `puppet resource`, which is the desired outcome.
+
+Rather than specifying `default_only: nil`, the most straightforward and self-evident way to mark a property as unsupported is to use the `_exclude: [my_platform]` YAML tag.
+
```

Comment:
It may not be obvious that `_exclude:` can be used for the entire provider or just for a single property. Would it help to include a couple of examples?